### PR TITLE
tests: add tests for dbrocket build

### PR DESCRIPTION
**Pull request by steven-mi**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import os
+
+import pytest as pytest
+
+from rocket.rocket import Rocket
+
+
+@pytest.fixture()
+def python_rocket():
```
Comment (jeanCarloMachado) [inline tests/conftest.py]: Lets type annotate everywhere possible, long term it really pays off for quality assurance. But in the short term it already pays off for readability :).

File: tests/test_rocket.py [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+    assert python_rocket.wheel_path
+
+
+def test_build_poetry(poetry_rocket):
+    poetry_rocket._build()
+    assert poetry_rocket.wheel_file
+    assert poetry_rocket.wheel_path
+
+
+def test_build_wrong_project_location():
```
Comment (jeanCarloMachado) [inline tests/test_rocket.py]: Could you please add a short sentence with what we are testing here (and all other tests, in english). On the long run tests without an uncontestable purpose tend to frustrate people and get removed :).

**Review by jeanCarloMachado — APPROVED** [review]
Thanks for paving the way of testing dbrocket. I liked the resources folder idea quite a lot :).

File: tests/test_rocket.py (hunk 2 above)
Comment (steven-mi) [reply tests/test_rocket.py]: 👍

File: tests/conftest.py (hunk 1 above)
Comment (steven-mi) [reply tests/conftest.py]: 👍

**Pull request merged**